includes/transformer/class-comment.php

```diff
@@ -53,31 +54,30 @@ public function change_wp_user_id( $user_id ) {
 	 * @return \Activitypub\Activity\Base_Object The ActivityPub Object.
 	 */
 	public function to_object() {
-		$comment = $this->wp_object;
-		$object  = parent::to_object();
-
-		$object->set_url( $this->get_id() );
-		$object->set_type( 'Note' );
+		$object = parent::to_object();
+
+		$content       = $this->get_content();
+		$at_replies    = '';
+		$reply_context = $this->extract_reply_context( array() );
+
+		foreach ( $reply_context as $acct => $url ) {
+			$at_replies .= sprintf(
+				'<a class="u-mention mention" href="%s">%s</a> ',
+				$url,
+				$acct
```

Comment:
```suggestion
		foreach ( $reply_context as $account => $url ) {
			$at_replies .= sprintf(
				'<a class="u-mention mention" href="%s">%s</a> ',
				$url,
				$account
```
Should `$url` and `$account` be escaped? Does this string end up being displayed?

Multiple mentions will be glued together without a space, not sure if that's on purpose?

Reply:
We have to escape them! good point, but I would love to stick with `$acct`, because it is the WebFinger that uses `acct:` as protocol, or we could use `$webfinger` instead.

Reply:
Makes sense!